plenum/server/consensus/consensus_shared_data.py

```diff
@@ -14,19 +14,20 @@ class ConsensusSharedData:
     TODO: Restore primary name from audit ledger instead of passing through constructor
     """
 
-    def __init__(self, name: str, validators: List[str], replica_index):
+    def __init__(self, name: str, inst_id):
```

Comment:
Please, don't forget type annotations (`inst_id: int`). Also, I still think that it's a good idea to inject initial validators list through constructor, especially given that you already have a code in replica for that.

Reply:
ok